ui/src/app/controllers/views/ControllerDetails/ControllerSummary/ServicesCard/ServiceStatus/ServiceStatus.test.tsx

```diff
@@ -0,0 +1,54 @@
+import { render, screen } from "@testing-library/react";
+
+import ServiceStatus from "./ServiceStatus";
+
+import {
+  ServiceName,
+  ServiceStatus as ServiceStatusName,
+} from "app/store/service/types";
+import { service as serviceFactory } from "testing/factories";
+
+it("correctly renders a running service", () => {
+  const service = serviceFactory({ status: ServiceStatusName.RUNNING });
+  render(<ServiceStatus service={service} />);
+
+  expect(screen.getByTestId("service-status-icon")).toHaveClass(
+    "p-icon--success"
+  );
+});
+
+it("correctly renders a degraded service", () => {
+  const service = serviceFactory({ status: ServiceStatusName.DEGRADED });
+  render(<ServiceStatus service={service} />);
+
+  expect(screen.getByTestId("service-status-icon")).toHaveClass(
+    "p-icon--warning"
+  );
+});
+
+it("correctly renders a dead service", () => {
+  const service = serviceFactory({ status: ServiceStatusName.DEAD });
+  render(<ServiceStatus service={service} />);
+
+  expect(screen.getByTestId("service-status-icon")).toHaveClass(
+    "p-icon--error"
+  );
```

Comment:
Might be useful to search for an accessible label instead of/on top of the class name.

```suggestion
  expect(screen.getByTestId("service-status-icon")).toHaveAccessibleName(
    "error"
  );
```